snapshot/lib/snapshot/runner.rb

```diff
@@ -123,12 +123,14 @@ def launch(language, locale, device_type, launch_arguments)
       File.write(File.join(prefix, "locale.txt"), locale || "")
       File.write(File.join(prefix, "snapshot-launch_arguments.txt"), launch_arguments.last)
 
-      Fixes::SimulatorZoomFix.patch
-      Fixes::HardwareKeyboardFix.patch
-
+      # Kill and shutdown all currently running simulators so that the following settings
+      # changes will be picked up when they are started again.
       Snapshot.kill_simulator # because of https://github.com/fastlane/snapshot/issues/337
       `xcrun simctl shutdown booted &> /dev/null`
 
+      Fixes::SimulatorZoomFix.patch
+      Fixes::HardwareKeyboardFix.patch
```

Comment:
Move the fixes to after the Simulator cleanup so that we only need to kill once.